client/src/components/modals/pick-entity-or-conversion.vue

```diff
@@ -1,58 +1,55 @@
 <template>
-  <v-dialog v-model="showModal" :max-width="width" hide-overlay>
-    <v-card class="pa-0 no-border">
+  <v-dialog v-model="showModal" :max-width="width" persistent>
+    <v-card class="pa-6 no-border pick-entity-card">
+      <v-row class="pl-1 mx-0">
+        <v-col cols="11">
+          <label>{{ locationText }}:</label>
+        </v-col>
+        <v-col cols="1">
+          <v-icon md color="primary" @click="showModal = false">mdi-close</v-icon>
+        </v-col>
+      </v-row>
       <template v-if="isConversion">
-        <v-card-text class="py-4 text-center">
-          <v-container fluid>
+        <v-card-text>
+          <v-container fluid class="px-4">
             <v-row>
-              <v-col cols="12" class="h5">Possible Conversions</v-col>
+              <v-col cols="12" class="copy-bold">Alterations</v-col>
             </v-row>
             <v-row v-for="(conversion, i) in conversionTypes" :key="'conv-'+i">
               <v-col cols="12" class="clickable-cell"
                      :id="conversion.value"
                      @click="chooseConversion(conversion)">
-                <v-tooltip bottom max-width="500" open-delay="500">
+                <v-tooltip bottom content-class="bottom-tooltip">
                   <template v-slot:activator="scope">
-                    <button v-on="scope.on" class="link-sm-sans-ul text-left">{{ conversion.desc }}</button>
+                    <button v-on="scope.on" class="link-sm-sans-ul entity-link">{{ conversion.desc }}</button>
                   </template>
-                  <ul class="ma-0 pa-0">
-                    <li class="no-bullet"
-                        v-html="conversion.blurb"
-                        :key="i + '-blurb'" />
-                  </ul>
+                  <div v-for="(text, i) in conversion.blurb" :key="i + '-blurb'">
+                    <span>{{ text }}</span>
+                  </div>
                 </v-tooltip>
               </v-col>
             </v-row>
           </v-container>
         </v-card-text>
       </template>
       <template v-else-if="!showSocietiesInfo">
-        <v-card-text class="py-4" style="display: flex; justify-items: center; width: 100%">
+        <v-card-text style="display: flex; justify-items: center; width: 100%">
```

Comment:
Don't change but FYI: there are Vuetify helper classes for this already.

https://vuetifyjs.com/en/styles/flex/ etc

Reply:
For sure, this was existing styling relocated, but a good idea to update where possible 👍

Reply:
Updated!